sprimber-engine/src/main/java/com/griddynamics/qa/sprimber/engine/Node.java

```diff
@@ -49,6 +49,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
     private Status status;
     private Phase phase;
     private Type type;
+    private final String adapterName;
```

Comment:
Why do you need it? Are you using it somewhere?

Reply:
yes, this is a part of unique node information, that will be utilized at least in reporting. that help to filter the nodes that belongs to particular adapter and be safe with internal tree structure that belongs uniquely to each adapter